### ISSUE TITLE
Home page template in LibGuides lacks H1 and H2 tags

### ISSUE THREAD

**Issue by tomkeays** [tomkeays/lemoyne#49]
WAVE accessibility audit complains about "Missing first level heading". H2 is also missing from the template. The reason is because the template -- only used on this single page -- omits the page title, which would be the H1 tag, as well as the sidebar navigation, which would be the H2 tags. Both of these would be supplied by the system if they were included in the template.

LibGuides has an accessibility setting that disallows H1 and H2 tags in the content boxes, so this page starts off with H3 and H4 tags. 

![image](https://user-images.githubusercontent.com/171037/68510405-722e7b00-0241-11ea-8a46-cfe44931e89f.png)
